keyboards/balloondogcaps/tr90pm/config.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2021 Balloondog
```

Comment:
This file can be deleted.

Reply:
According to the VIA documentation I need that file.

Source: https://www.caniusevia.com/docs/configuring_qmk

Reply:
`MANUFACTURER` and `PRODUCT` will be generated from the info.json, and `DESCRIPTION` should be removed as it doesn't do anything. So no, you do not need this file.